src/makeInfinityAdder.js

```diff
@@ -1,12 +1,23 @@
 'use strict';
 
-/***
- *
- * @return {function}
- */
-
 function makeInfinityAdder() {
-  // write code here
+  let result = 0;
+
+  const add = a => {
+    if (a === undefined) {
+      const output = result;
+
+      result = 0;
+
+      return output;
+    }
```

Comment:
just provide a default value for a - instead of checking for undefined
```suggestion
  const add = (a = 0) => {
    ...
```

Reply:
@IoghiStudio to address your concern - if your code passes tests, you can be approved.
If you don't agree with review comments, you can tell us why.
If you want to get approved - as said earlier. You can be.